source/faq.txt

```diff
@@ -56,9 +56,13 @@ Which connection string options does ``mongosync`` allow?
 ---------------------------------------------------------
 
 ``mongosync`` requires :ref:`readConcern: "majority" <read-concern>`
-and :ref:`writeConcern: "majority" <write-concern>`. ``mongosync`` 
-overwrites any other ``readConcern`` or ``writeConcern`` and uses
-``"majority"`` instead.
+and :ref:`writeConcern: "majority" <write-concern>`. If the
+``readConcern`` or ``writeConcern`` is not ``majority``, ``mongosync`` 
+returns an error:
+
+.. code-block:: shell
+
+   Invalid URI option, write/read concern must be majority
```

Comment:
My apologies, this is my miscommunication: if the invalid options is `write concern`, then the error will be:

```
Invalid URI option, write concern must be majority
```

if the invalid option is `read concern`, the error will be

```
Invalid URI option, read concern must be majority
```

The current state of the documentation looks good to me, but I'm not sure if the clarification would change how you phrase it.

Reply:
If you still want to go ahead with this change, let me know and I can approve it. Sorry for the confusion again!

Reply:
No worries, I've updated the text